diff --git a/chunk-tgg3x73e.js b/chunk-tgg3x73e.js
index 4989e111b701c3509f96cfb4a2fbccb273eb4f22..900e355318661f9f731d86e77660f5e49cc64486 100644
--- a/chunk-tgg3x73e.js
+++ b/chunk-tgg3x73e.js
@@ -159,7 +159,7 @@ class ErrorBoundary extends React.Component {
 
 // src/reconciler/reconciler.ts
 import ReactReconciler from "react-reconciler";
-import { ConcurrentRoot } from "react-reconciler/constants";
+import { ConcurrentRoot } from "react-reconciler/constants.js";
 
 // src/reconciler/host-config.ts
 import { TextNodeRenderable as TextNodeRenderable2 } from "@opentui/core";
@@ -244,7 +244,7 @@ var package_default = {
 
 // src/reconciler/host-config.ts
 import { createContext as createContext2 } from "react";
-import { DefaultEventPriority, NoEventPriority } from "react-reconciler/constants";
+import { DefaultEventPriority, NoEventPriority } from "react-reconciler/constants.js";
 
 // src/utils/id.ts
 var idCounter = new Map;
