diff --git a/chunk-rm53pqj2.js b/chunk-rm53pqj2.js
index 788691fabed796287b13eb1cc6b2cfb71369de36..d285b07cc1f313904cff1badfaca06bf0d2e0b7d 100644
--- a/chunk-rm53pqj2.js
+++ b/chunk-rm53pqj2.js
@@ -159,7 +159,7 @@ class ErrorBoundary extends React.Component {
 
 // src/reconciler/reconciler.ts
 import ReactReconciler from "react-reconciler";
-import { ConcurrentRoot } from "react-reconciler/constants";
+import { ConcurrentRoot } from "react-reconciler/constants.js";
 
 // src/reconciler/host-config.ts
 import { TextNodeRenderable as TextNodeRenderable2 } from "@opentui/core";
@@ -244,7 +244,7 @@ var package_default = {
 
 // src/reconciler/host-config.ts
 import { createContext as createContext2 } from "react";
-import { DefaultEventPriority, NoEventPriority } from "react-reconciler/constants";
+import { DefaultEventPriority, NoEventPriority } from "react-reconciler/constants.js";
 
 // src/utils/id.ts
 var idCounter = new Map;
